symphony/app/fbcnms-projects/hub/app/components/__generated__/CreateServiceMutation.graphql.js

```diff
@@ -4,7 +4,7 @@
  *
  **/
 
- /**
```

Comment:
This file is generated so it shouldn't be changed here. Maybe something in your editor did this automatically? Eslint ignores it, so it won't be an issue.

Reply:
Fixed